tests/phpunit/tests/ajax/wpAjaxWpCompressionTest.php

```diff
@@ -145,7 +145,7 @@ public function test_set_yes() {
 		}
 
 		// Check the site option is not changed due to lack of nonce.
-		$this->assertSame( 0, get_site_option( 'can_compress_scripts' ) );
+		$this->assertSame( false, get_site_option( 'can_compress_scripts' ) );
```

Comment:
This can just use `assertFalse()` instead.

Reply:
Thank you for bringing this up. The changes were reverted because they caused backward compatibility issues, and the code was adjusted accordingly. Thanks!